hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneKey.java

```diff
@@ -179,6 +207,16 @@ public ReplicationConfig getReplicationConfig() {
     return replicationConfig;
   }
 
+  @JsonIgnore
+  public Long getObjectID() {
```

Comment:
I didn't want to change the output displayed by key info command. It can be argued either way whether they should be displayed or not I think, as they are kind of internal "not normal client facing things".